.github/workflows/build-and-test.yml

```diff
@@ -14,8 +14,8 @@ jobs:
         # This is an undocumented way of creating a matrix of multiple values, see:
         # https://stackoverflow.com/questions/66025220/paired-values-in-github-actions-matrix.
         build: [
-            {solution: Hastlayer.SDK.Client.sln, machine-type: "['ubuntu-22.04']", test-disable: true},
-            {solution: Hastlayer.SDK.Linux.sln, machine-type: "['ubuntu-22.04']", test-disable: true},
+            {solution: Hastlayer.SDK.Client.sln, machine-type: "['ubuntu-22.04']", test-disable: false},
```

Comment:
Why are we running tests for the Client project?

Reply:
There are tests you can access from the client solution too, so why not?

Reply:
We're already running them in the full solution.